### ISSUE TITLE
Evaluate workflow if chatbot is shown on front page

### ISSUE THREAD

**Issue by krumware** [CovidSC/covidsc-web#52]
# Enhancement

## Desired Functionality

It is requested that the chatbot be shown on the front page, but this may require some re-evaluation as to the user workflow. We _can_ customize the way the chatbot launcher works, it doesn't have to launch from the "corner bubble".

Do we need to revisit any workflow to do this? We would have 4 action items on the front page if all features are implemented (notices, check symptoms, get resources, chatbot). 

I will get the _different_ symptoms bot rolling on the dev site this morning to help evaluate if we have 2 different bots, how that changes things as well.

## Current Functionality

Today, the chatbot opens and launches on the Resources Page, to avoid mental conflicts with the home page's `check symptoms` and `find resources` actions.
https://www.covidsc.com/resources

Additional documentation of the chatbot is in #36 and https://dev.landbot.io/docs/?content=guide_landbot_embed

## Other remarks

Opinion:
I still think all resources and information supplied by the chatbot needs to correlate back to actual information listed on the site, so that users don't have to talk to the bot after they know what they are looking for.

**Comment by kbakeann** [comment]
Okay so I'm thinking maybe we remove the "check symptoms" button and just have the "find resources" button. Then we add the chat bot bubble to the homepage so that users can check their symptoms there. 

I also think having a resource page that lists symptoms is a good idea. This way if a user finds themselves in the resources and they want to know the symptoms of Covid19 without talking to a bot they can just read through them. 

This would remove the confusion of having "check symptoms" and a chat bubble that do the same thing on the same page, while still offering symptom related resources to those who would like to use the bot and those who don't.

**Comment by krumware** [comment]
If we remove the check symptoms or resources buttons, I'm worried that the purpose or functions of the application wouldn't be clear to a user who hasnt already been educated about it

**Comment by kbakeann** [comment]
I agree, I'm currently running some A/B testing to test this theory. Will update with results

**Comment by kbakeann** [comment]
So I tested four screens with 5 users. Here are the results:


## Screen 1
This screen has the "Check symptoms" and "Find resources" buttons.
<img width="405" alt="Screen Shot 2020-04-09 at 1 04 11 PM" src="https://user-images.githubusercontent.com/33105088/78925510-8df4b200-7a69-11ea-9be4-0bcb60ca8a23.png">

### Results: 
All 5 users would choose "Check symptoms" button if they wanted to find out if they had Covid19.

## Screen 2
This screen has the "Check symptoms" and "Find resources" buttons as well as the chatbot icon in the lower right hand corner.
<img width="408" alt="Screen Shot 2020-04-09 at 1 03 19 PM" src="https://user-images.githubusercontent.com/33105088/78925632-c0061400-7a69-11ea-8ea7-e3d6cea16fe7.png">

### Results: 
All 5 users would select "Check symptoms" button to find out if they have Covid19.

## Screen 3
This screen only has the "Find resources" button and the chat bot icon in the lower right hand corner.
<img width="409" alt="Screen Shot 2020-04-09 at 1 08 21 PM" src="https://user-images.githubusercontent.com/33105088/78925757-f774c080-7a69-11ea-9339-456d57809a40.png">

### Results: 
4 out of 5 users would select "Find Resources" to find out if they have Covid19. Only one user would select the chat bot.

## Screen 4
This screen has the "Find resources" button, the chatbot icon, and an informative bit of text telling them to use the chatbot to find out if they have Covid19.
<img width="407" alt="Screen Shot 2020-04-09 at 1 03 27 PM" src="https://user-images.githubusercontent.com/33105088/78925946-3dca1f80-7a6a-11ea-9a26-14377261917f.png">

### Results:
4 out of 5 users would select the chat bot icon to find out if they have Covid19. One user continued to select the "Find resources" button.


## Notes:

1. The majority of these users would not trust a chatbot to tell them if they have Covid19. They would rather read a list of symptoms and determine that for themselves.

2. All users stated that they would avoid using a chatbot if they have the option. 

3. One user said they completely skipped over the informative text in `Screen 4` and continued on to select "Find resources"

4. One user said `Screen 4` makes it feel as if someone is there for them by offering the text at the top of the page. Whereas `Screens 1 & 2` were just actions, and `Screen 3` was just confusing.


## Conclusion: Most users do not like using chatbots and will avoid them at all costs. If we do move the chatbot bubble to the main screen it does not make sense to also include the "Check symptoms" button because users will use the button every time. And if we do add the bubble to the homepage we would need supportive text leading the user in that direction as they will not know what the bubble is for otherwise.

@krumware

**Comment by kbakeann** [comment]
Okay so after reviewing the results of the user test with Colin we came to an agreement on how this should pan out. 

The Check Symptoms button should take you to a Check symptoms page where the chatbot for reporting symptoms will automatically be open and will start. This may change in the future but this is how it will work for now. This chatbot _only_ deals with being sick/checking symptoms.

The Find Resources page should have some text at the top that states something along the lines of "Don't know where to start? Try our chatbot to figure out which resources are right for you". This will help point the user in the direction of the resources chatbot in the lower right-hand corner. This chatbot will offer financial resources and resources not relating to symptoms or being sick.

We may also add some text to the homepage explaining the importance of reporting symptoms and generally stating our mission as CovidSC. This will outline the purpose of the site and hopefully get people to share the site with their friends.
